File: WebListIntent/app/src/main/java/com/dstrube/weblistintent/controller/ContentFragmentActivity.java
package com.dstrube.weblistintent.controller;

import com.dstrube.weblistintent.view.ContentFragment;

import android.content.Intent;
import android.os.Bundle;
import androidx.fragment.app.FragmentActivity;

public class ContentFragmentActivity extends FragmentActivity {
	@Override
	protected void onCreate(Bundle arg0) {
		super.onCreate(arg0);
		
		ContentFragment contentFragment = new ContentFragment();
        
        Intent i = this.getIntent();
        String link = i.getExtras().getString("link");
        
        contentFragment.init(link);
        getFragmentManager().beginTransaction().add(android.R.id.content, contentFragment).commit();
	}
}

File: kotlin/fix/WebListIntent/app/src/main/java/com/dstrube/weblistintent/controller/ContentFragmentActivity.kt
package com.dstrube.weblistintent.controller

//import android.content.Intent.getIntent
//import android.content.Intent
import android.os.Bundle
import android.os.PersistableBundle
import android.support.v4.app.FragmentActivity;
import com.dstrube.weblistintent.view.ContentFragment

class ContentFragmentActivity : FragmentActivity() {

    override fun onCreate(savedInstanceState: Bundle?, persistentState: PersistableBundle?) {
        super.onCreate(savedInstanceState, persistentState)//super.onCreate(arg0)

        val contentFragment = ContentFragment()

        val i = this.getIntent()
        val link = i.getExtras()!!.getString("link")

        contentFragment.init(link)
        getFragmentManager().beginTransaction().add(android.R.id.content, contentFragment).commit()
    }
}